$MirOS: ports/devel/cvs/patches/patch-diff_side_c,v 1.2 2007/03/10 23:53:48 tg Exp $
--- diff/side.c.orig	Thu Jun 18 16:56:53 1998
+++ diff/side.c	Sat Mar 10 23:14:14 2007
@@ -21,6 +21,8 @@ and this notice must be preserved on all
 
 #include "diff.h"
 
+__RCSID("$MirOS: ports/devel/cvs/patches/patch-diff_side_c,v 1.2 2007/03/10 23:53:48 tg Exp $");
+
 static unsigned print_half_line PARAMS((char const * const *, unsigned, unsigned));
 static unsigned tab_from_to PARAMS((unsigned, unsigned));
 static void print_1sdiff_line PARAMS((char const * const *, int, char const * const *));
@@ -122,7 +124,7 @@ print_half_line (line, indent, out_bound
 	  break;
 
 	case '\b':
-	  if (in_position != 0 && --in_position < out_bound)
+	  if (in_position != 0 && --in_position < out_bound) {
 	    if (out_position <= in_position)
 	      /* Add spaces to make up for suppressed tab past out_bound.  */
 	      for (;  out_position < in_position;  out_position++)
@@ -133,6 +135,7 @@ print_half_line (line, indent, out_bound
 		cc = c;
 		write_output (&cc, 1);
 	      }
+	     }
 	  break;
 
 	case '\f':
